src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -517,7 +517,10 @@ function getSeries(series: any, colorPalette: any = []) {
 
 function getHeatMapDataConfiguration(chartOptions: any) {
     const data = chartOptions.data || EMPTY_DATA;
-    const series = data.series;
+    const series = data.series.map((item: any) => ({
+        ...item,
+        borderWidth: 0
```

Comment:
can it be set directly in heatMapConfiguration.ts staticly instead of iterating data here?